[develop] Change file paths to EPIC data locations and add fix_gsi var

EdwardSnyder-NOAA opened the pull request:
## DESCRIPTION OF CHANGES: 
<!-- One or more paragraphs describing the problem, solution, and required changes. -->
Updated the data file paths to point from SRW account locations to EPIC role account locations, so that EPIC can have better control and management of the SRW App data files. 

Also added the FIXgsi var.

### Type of change
<!-- Please delete options that are not relevant. Add an X to check off a box. -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## TESTS CONDUCTED: 
<!-- Explicitly state what tests were run on these changes, or if any are still pending (for README or other text-only changes, just put "None required"). Make note of the compilers used, the platform/machine, and other relevant details as necessary. For more complicated changes, or those resulting in scientific changes, please be explicit! -->
<!-- Add an X to check off a box. -->

- [x] hera.intel
- [x] orion.intel
- [x] cheyenne.intel
- [ ] cheyenne.gnu
- [x] gaea.intel
- [x] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [ ] fundamental test suite
- [x] comprehensive tests (specify *which* if a subset was used)
nco_grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_HRRR
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16
community_ensemble_2mems
grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v16
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_HRRR
MET_verification

## DEPENDENCIES:
<!-- Add any links to external PRs (e.g. regional_workflow and/or UFS PRs). For example:
- ufs-community/regional_workflow/pull/<pr_number>
- ufs-community/UFS_UTILS/pull/<pr_number>
- ufs-community/ufs-weather-model/pull/<pr_number> -->

## DOCUMENTATION:
<!-- If this PR is contributing new capabilities that need to be documented, please also include updates to the RST files (docs/UsersGuide/source) as supporting material. -->
Updated RunSRW and ContainerQuickstart rst files. The files listed paths to the older location and have been updated to point to the EPIC location.

## ISSUE: 
<!-- If this PR is resolving or referencing one or more issues, in this repository or elsewhere, list them here (Remember, issues must always be created before starting work on a PR branch!). For example, "Fixes issue mentioned in #123" or "Related to bug in https://github.com/ufs-community/other_repository/pull/63" -->
This PR addresses the lack of FIXgsi directory in the develop branch as described in issue #672 

## CHECKLIST
<!-- Add an X to check off a box. -->
- [x] My code follows the style guidelines in the Contributor's Guide
- [x] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain).
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

## LABELS (optional): 
<!-- If you do not have permissions to add labels to your own PR, request that labels be added here. 
Add an X to check off a box. Delete any unnecessary labels. -->
A Code Manager needs to add the following labels to this PR: 
- [x] Work In Progress
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] release
- [ ] high priority
- [ ] run_ci
- [ ] run_we2e_fundamental_tests
- [ ] run_we2e_comprehensive_tests
- [ ] Needs Cheyenne test 
- [ ] Needs Jet test 
- [ ] Needs Hera test 
- [ ] Needs Orion test 
- [ ] help wanted

## CONTRIBUTORS (optional): 
<!-- If others have contributed to this work aside from the PR author, list them here -->
@natalie-perlin for her help moving data, documenting, and testing the changes.

EdwardSnyder-NOAA committed:
updated file paths to EPIC data locations

EdwardSnyder-NOAA committed:
updated file paths to EPIC data locations

EdwardSnyder-NOAA committed:
fixed table

EdwardSnyder-NOAA committed:
update noaacloud paths

EdwardSnyder-NOAA committed:
Merge branch 'ufs-community:develop' into feature/epic-data-locations

EdwardSnyder-NOAA committed:
added FIXgsi var

EdwardSnyder-NOAA committed:
Merge branch 'feature/epic-data-locations' of https://github.com/EdwardSnyder-NOAA/ufs-srweather-app into feature/epic-data-locations

natalie-perlin commented on ush/machine/gaea.yaml:
```diff
@@ -18,16 +18,17 @@ platform:
+  FIXshp: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/NaturalEarth
 data:
   ics_lbcs:
     FV3GFS: /lustre/f2/dev/Mark.Potts/EPIC/SRW/model_data/FV3GFS/${yyyymmdd}${hh}
```
This line may need to be changed, too, as it points to the old location.
Suggesting:

```
  FV3GFS:
    nemsio: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/input_model_data/FV3GFS/nemsio/${yyyymmdd}${hh}
    grib2: /lustre/f2/dev/role.epic/contrib/UFS_SRW_data/develop/input_model_data/FV3GFS/grib2/${yyyymmdd}${hh}
```

natalie-perlin commented on ush/machine/noaacloud.yaml:
```diff
@@ -14,17 +14,18 @@ platform:
+  FIXgsi: /contrib/EPIC/UFS_SRW_data/develop/fix/fix_gsi
   EXTRN_MDL_DATA_STORES: aws nomads
 data:
   ics_lbcs:
```
Suggest to modify the subsequent line to:

  ```
  FV3GFS:
       nemsio: /contrib/EPIC/UFS_SRW_data/develop/input_model_data/FV3GFS/nemsio/${yyyymmdd}${hh}
       grib2: /contrib/EPIC/UFS_SRW_data/develop/input_model_data/FV3GFS/grib2/${yyyymmdd}${hh}
```

EdwardSnyder-NOAA replied:
done.

natalie-perlin reviewed (commented):
It would be good to get any comments or suggesions on how to deal with the old locations specified under "data" section of the machine.yaml file

EdwardSnyder-NOAA committed:
updated data FV3GFS file pat

EdwardSnyder-NOAA commented:
@natalie-perlin - the paths found under the "data" section are the default locations of the data files for each platform. Setting this variable to true: `USE_USER_STAGED_EXTRN_FILES` will trigger the generation script to look at our EPIC location via this variable:
`EXTRN_MDL_SOURCE_BASEDIR_*`. 

I did change the two "data" paths that were pointing to older locations to our EPIC ones. I don't think any further modifications are needed.

natalie-perlin reviewed (approved):
@EdwardSnyder-NOAA - thank you for taking care of the new FIXgsi data on the platforms!

gsketefian commented:
@EdwardSnyder-NOAA FYI In my PR #683 I added new data on Hera.  Details:
1) Added the directory `/scratch2/BMC/det/UFS_SRW_App/develop/output_data/fcst_det/RRFS_CONUS_25km/rrfs.20190615`.   This just contains some symlinks to enable running the workflow in nco mode.  The symlinks are in the subdirectory `00` and point to files in `/scratch2/BMC/det/UFS_SRW_App/develop/output_data/fcst_det/RRFS_CONUS_25km/2019061500/postprd`.
2) Added the directory `/scratch2/BMC/det/UFS_SRW_App/develop/output_data/fcst_ens`.  This is to enable ensemble verification from staged data.  There are actual data files (in the subdirectories `RRFS_CONUS_25km/2019061500/mem[001|002]/postprd`) for running in community mode and symlinks to those files (in the subdirectory `rrfs.20190615/00`) for running in nco mode.

So these need to be copied over to the EPIC locations and the variable `TEST_VX_FCST_INPUT_BASEDIR` in `ush/machines/hera.yaml` updated accordingly (please also see my version of this file since the variable now contains some jinja logic that needs to be retained).  Thanks!

MichaelLueken reviewed (approved):
@EdwardSnyder-NOAA These changes look good to me!  I will go ahead and approve these changes now, but as @gsketefian noted, please make sure that the new data on Hera has been added to the EPIC maintained data locations.

EdwardSnyder-NOAA committed:
updated logic in TEST_VX_FCST_INPUT_BASEDIR var

EdwardSnyder-NOAA commented:
@gsketefian & @MichaelLueken - I've updated the data in our EPIC data locations as described in Gerard's comment and updated the jinja logic in the hera.yaml.

gsketefian commented:
@EdwardSnyder-NOAA Your change looks good to me.  Thanks.

christinaholtNOAA reviewed (approved):
LGTM

MichaelLueken commented:
@EdwardSnyder-NOAA I have manually reviewed the logs in the Jenkins workspaces on the various machines to make sure that all tests successfully passed (since Jenkins is currently down due to migration to a new site).  All Jenkins tests have passed.  Since you have the necessary approvals, I will go ahead and merge this work now.  I have also removed your issue #672 from being closed, as it looks like you will need to add supported grids for RRFS.  Thanks!
